addons/html_builder/static/src/snippets/snippet_service.js

```diff
@@ -88,9 +88,7 @@ export class SnippetModel extends Reactive {
 
         this.dialog.add(ConfirmationDialog, {
             title: _t("Install %s", snippet.title),
-            body: markup(
-                `${escape(bodyText)}\n<a href="${linkUrl}" target="_blank">${escape(linkText)}</a>`
-            ),
+            body: markup`${bodyText}\n<a href="${linkUrl}" target="_blank">${linkText}</a>`,
```

Comment:
Slightly out of scope: is `ConfirmationDialog` applying nl2br?

This is strange to give `\n` instead of `<br>` directly inside HTML, it feels like a feature that should only be applied on text content and not when HTML is given (as `\n` are supposed to be simple blank char in HTML, not new lines).